### DISCUSSION TITLE
Applying Dynamic Prompt to SD Upscale (Img2Img)?

### DISCUSSION THREAD

**Discussion by owenprescott** [adieyal/sd-dynamic-prompts#401] [Q&A]
I have uploaded a picture of a city. I want to create several similar (~65%) variations of the city using my dynamic prompt wildcards and then upscale using the SD upscale extension (in one go (if possible)). 

The problem I'm running into is that my random wildcards are being applied to each tile during upscale, making the image incoherent. I'm hoping someone has experience SD upscaling and can maybe recommend a setting to try, or maybe it's just not possible? Fyi I'm using "Unlink seed from prompt" which I suspect is intended for this purpose hoverever the upscale process messes it up (as far as I can tell).

Not sure if this is more relevant to SD upscale or Dynamic Prompts, any feedback would be great.

![2023-04-24_03h43_29](https://user-images.githubusercontent.com/3827398/233890388-9fe4e94e-7cd3-42bc-b7d4-7534823735bf.jpg)

**Comment by akx** [comment]
I have to say I like the psychedelic collage effect here 😁 

For upscaling a single image, you can grab the actual generated prompt via the PNG Info tab and paste it as the img2img prompt when resizing, but that will of course be a manual step, so batch-upscaling with that won't do...
